tools/testsys/src/run.rs

```diff
@@ -266,55 +309,47 @@ pub(crate) struct Image {
     pub(crate) id: String,
 }
 
+/// This is a CLI parsable version of `testsys_config::TestsysImages`
 #[derive(Debug, Parser)]
 pub(crate) struct TestsysImages {
```

Comment:
`CliConfig` has a 1 to 1 mapping with a different struct name (`GenerictVariantConfig`), would it make sense to do the same for this struct and either rename this or the counterpart in `testsys_config`?

Reply:
I don't think it quite works out, because the images available from the config can be split into 2 separate sections, controller and agents. I may add the controller image to the config file in a later pr.